manageiq-operator/pkg/apis/manageiq/v1alpha1/manageiq_types.go

```diff
@@ -38,6 +39,10 @@ type ManageIQSpec struct {
 	// +optional
 	ImagePullSecret string `json:"imagePullSecret"`
 
+	// StorageClass name that will be used by manageiq data stores
+	// +optional
+	StorageClassName string `json:"storageClassName"`
```

Comment:
Do you think we will need separate storage classes for the separate PVCs? I'm not sure what the difference between classes would be.

Reply:
Like `ImagePullSecret`, I want one cluster use one `StorageClassName` for the ManageIQ Operator.